sdk/cosmos/azure-cosmos-spark_3_2-12/src/test/scala/com/azure/cosmos/spark/TransientIOErrorsRetryingIteratorITest.scala

```diff
@@ -2,7 +2,7 @@
 // Licensed under the MIT License.
 package com.azure.cosmos.spark
 
-import com.azure.cosmos.CosmosItemSerializer
+import com.azure.cosmos.{CosmosItemSerializer, CosmosItemSerializerNoExceptionWrapping}
```

Comment:
nit: `CosmosItemSerializer` would be unused.

Reply:
Fixed